### PR TITLE
Closes #3 - Create initial index.html markup

### LINKED ISSUE #3: Create initial index.html markup

### PR DESCRIPTION
**Pull request by koldovsky**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    
+</body>
+</html>
```
Comment (romanolytvyn) [inline index.html]: Please add empty string after code

Comment (AndriyTykhan) [inline index.html]: Pleace add last line

Comment (machikharostyk) [inline index.html]: Please add new empty line at the end

**Review by OBats — CHANGES_REQUESTED** [review]
Please add a new line at the end

**Review by machikharostyk — CHANGES_REQUESTED** [review]
Please add new empty line at the end

File: index.html (hunk 1 above)
Comment (Yuri-Pristay) [inline index.html]: Please add new line at the end of file.

Comment (rostyslav-vasyliuk) [inline index.html]: Better to add new line

**Review by Roman6917 — COMMENTED** [review]
Please add empty line

File: index.html (hunk 1 above)
Comment (tarasandrusiv) [inline index.html]: Please add last empty line

**Review by romanolytvyn — COMMENTED** [review]
Please see a comment

**Review by Yuri-Pristay — CHANGES_REQUESTED** [review]

**Review by AndriyTykhan — CHANGES_REQUESTED** [review]
Please add last empty line at the end

**Review by tarasandrusiv — APPROVED** [review]
Please add last empty line

**Review by anzhelikavelychko — CHANGES_REQUESTED** [review]
Add new line

**Review by tarasandrusiv — CHANGES_REQUESTED** [review]
please add new line

**Review by rostyslav-vasyliuk — CHANGES_REQUESTED** [review]
Better to add new line

File: index.html (hunk 1 above)
Comment (romanolytvyn) [inline index.html]: Please add empty line after code

**Review by romanolytvyn — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Roman6917) [inline index.html]: add empty line

**Review by Roman6917 — CHANGES_REQUESTED** [review]
add empty line

**Review by OBats — APPROVED** [review]

**Review by romanolytvyn — APPROVED** [review]

**Review by AndriyTykhan — APPROVED** [review]

**Review by anzhelikavelychko — APPROVED** [review]

**Review by machikharostyk — APPROVED** [review]

**Review by rostyslav-vasyliuk — APPROVED** [review]

**Review by Yuri-Pristay — APPROVED** [review]

**Review by Roman6917 — APPROVED** [review]

**Review by tarasandrusiv — APPROVED** [review]

**Pull request merged**